test/unit/conftest.py

```diff
@@ -200,5 +203,6 @@ def prepare_basic_config(tmpdir):
     BasicConfig.MAIL_FROM = ""
     BasicConfig.ALLOWED_HOSTS = ""
     BasicConfig.DENIED_HOSTS = ""
-    BasicConfig.CONFIG_FILE_FOUND = False
+    BasicConfig.CONFIG_FILE_FOUND = True
```

Comment:
For now setting this as `True`, but I think in the future we might need a fixture that creates the config and also prepares the DB and `.autosubmitrc` file.